addons/hr_holidays/static/src/js/time_off_calendar.js

```diff
@@ -85,16 +85,11 @@ odoo.define('hr_holidays.dashboard.view_custo', function(require) {
         _onNewTimeOff: function () {
             let self = this;
 
-            let domain = [
-                ['name', '=', 'hr.leave.view.form.dashboard.new.time.off'],
-                ['model', '=', 'hr.leave']
-            ];
-
             self._rpc({
-                model: 'ir.ui.view',
-                method: 'search',
-                args: [domain],
-            }).then(function(ids) {
+                model: 'hr.leave',
+                method: 'get_new_time_off_view_ids',
+                args: [[]],
+            }).then(function (ids) {
```

Comment:
```suggestion
                route: '/web/action/load',
                params: {action_id: 'hr_holidays.hr_leave_action_my_request_view_form'}
            }).then(function (action) {
```
Could we use the action to retrieve the id instead of creating a new method?